src/pfe/portal/modules/Project.js

```diff
@@ -334,12 +336,21 @@ module.exports = class Project {
   }
 
   toJSON() {
-    // Strip out fields we don't want to save in the .inf file or log in debug if we print the project object.
+    // Strip out fields we don't want to attempt to turn into JSON.
     // (This is our guard against trying to write circular references.)
-    const { capabilitiesReady, detailedAppStatus, logStreams, loadInProgress, loadConfig, operation, ...filteredProject } = this;
+    const { logStreams, loadInProgress, loadConfig, operation, ...filteredProject } = this;
     return filteredProject;
   }
 
+  filterNonPersistentFields(key, value) {
```

Comment:
This function adds a class-methods-use-this eslint warning. Should it be a helper function outside of the project class?